graphql-schema.ts

```diff
@@ -13,7 +13,7 @@ export const typeDefs = gql`
     relayInformation: RelayInformation
     channel(name: String!): Channel
     deleted_events: [String!]!
-    members: [String!]!
+    members: [SpaceMember!]!
```

Comment:
Let's not continue the graphql route. It's model is fundementally incompatible with event sourcing with sigs.

Reply:
Okay, I will remove the members section tomorrow.